simplq/src/components/pages/Admin/Item.jsx

```diff
@@ -47,48 +46,48 @@ function Item(props) {
         setDidNotify(true);
       })
       .catch((err) => {
+        setNotifying(false);
         handleApiErrors(err);
       });
     // Notify user of error TODO
   };
   const onDeleteClick = () => {
-    TokenService.remove(tokenId).catch((err) => {
-      handleApiErrors(err);
-    });
-    props.removeItemHandler(tokenId); // TODO Should delete from list only ofter successfull deletion from db
+    props.removeItemHandler(tokenId);
```

Comment:
Is removing from the tokenService not required anymore?

Reply:
yes, removeItemHandler will call the service, and on success will remove the token from the state as well